File: C++/BubbleSort.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int *arr;
    int n,temp;
    cout<<"Enter number of elements to be entered in the array:"<<endl;
    cin>>n;
    arr=new int[n];
    cout<<"Enter "<<n<<" elements to be sorted:"<<endl;
    for(int i=0;i<n;i++)
    cin>>arr[i];
    for(int step=0;step<n-1;step++)
    {
        for(int i=0;i<n-step;i++)
        {
            if(arr[i]>arr[i+1])
            {
                temp=arr[i+1];
                arr[i+1]=arr[i];
                arr[i]=temp;
            }
        }
    }
    cout<<"The sorted array is:"<<endl;
    for(int i=0;i<n;i++)
    cout<<arr[i]<<endl;
}

File: C#/Bubble_sort.cs
using System;
namespace BubbleSort {
   class MySort {
      static void Main(string[] args) {
         Console.WriteLine("Enter the no. of element in array");
         int to_ele =  Int32.Parse(Console.ReadLine());
         int []arr=new int[to_ele];
         Console.WriteLine("Enter the value of elements");
         for (int i=0;i<arr.Length;i++){
             arr[i] = Int32.Parse(Console.ReadLine());
         }   
         int temp;
         for (int j = 0; j <= arr.Length - 2; j++) {
            for (int i = 0; i <= arr.Length - 2; i++) {
               if (arr[i] > arr[i + 1]) {
                  temp= arr[i + 1];
                  arr[i + 1] = arr[i];
                  arr[i] = temp;
               }
            }
         }
         Console.WriteLine("Sorted:");
         foreach (int p in arr)
            Console.Write(p + " ");
         Console.Read();
      }
   }
}
